deployment-guide/deployment-guides/iis-deployment-installation.md

```diff
@@ -62,118 +124,97 @@ Set-ItemProperty "IIS:\AppPools\$applicationPoolNameSSO" -name processModel.iden
 Set-ItemProperty "IIS:\AppPools\$applicationPoolNameSSO" -name processModel.userName -Value $username
 ```
 
-{% hint style="info" %}
-You may use an alternate application pool name if you prefer.
-{% endhint %}
-
 ## Create the application directories
 
-1. Unzip the **"Cinchy vX.X"** application package (for the Cinchy Platform) that you downloaded from the [Releases Table](https://cinchy.net/Cinchy/Tables/1477?rowHeight=Expanded) into your C drive. This will create 2 directories, C:\Cinchy and C:\CinchySSO. Ensure your application pool accounts has read and execute access to these directories (default accounts are IIS AppPool\CinchyWeb and IIS AppPool\CinchySSO).
-2. Run the below commands in the Administrator instance of PowerShell to create directories for the application logs. Ensure your application pool account has write access to these directories.
+1. Download and unzip the **"Cinchy vX.X"** application package from the
```

Comment:
ensure consistency. We specified that it should be "Cinchy vX.X." to match the previous reference of "v5.6.2."